packages/instrumentation-dataloader/src/instrumentation.ts

```diff
@@ -54,7 +61,8 @@ export class DataloaderInstrumentation extends InstrumentationBase<DataloaderIns
       new InstrumentationNodeModuleDefinition(
         MODULE_NAME,
         ['>=2.0.0 <3'],
-        dataloader => {
+        dataloaderPkg => {
+          const dataloader = extractModuleExports(dataloaderPkg);
```

Comment:
Very minor nitpicking but this doesn't match the naming used most other places (`module`/`moduleExports`)
```suggestion
        module => {
          const dataloader = extractModuleExports(module);
```